src/python/pants/backend/core/targets/resources.py

```diff
@@ -19,10 +19,12 @@ class Resources(Target):
   and friends API. In the ``jar`` goal, the resource files are placed in the resulting `.jar`.
   """
 
-  def __init__(self, address=None, payload=None, sources=None, **kwargs):
+  def __init__(self, address=None, payload=None, sources=None, use_jar=True, **kwargs):
     """
     :param sources: Files to "include". Paths are relative to the
       BUILD file's directory.
+    :param use_jar: Whether or not the resources will be zipped into a jar. This flag is ignored
```

Comment:
Should make it clear that the flag doesn't affect bundle/binary (when things are always jar'd.)